src/Sylius/Bundle/AdminBundle/Resources/assets/bootstrap.js

```diff
@@ -25,3 +26,4 @@ app.register('live', LiveController);
 app.register('slug', SlugController);
 app.register('taxon-slug', TaxonSlugController);
 app.register('product-attribute-autocomplete', ProductAttributeAutocomplete);
+app.register('move-positions', MovePositionsController);
```

Comment:
The name is weird.
Maybe `SavePositions`, `UpdatePositions`, `ChangePositions` or a bit less technical `Reposition`?